File: WebApp/Models/PriceList.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class PriceList
    {
        public int Id { get; set; }
       
        public string From { get; set; }
        public string To { get; set; }
        [Required]
        [Range(0, 15000)]
        public float TicketPrice { get; set; }
        public int PassengerTypeId { get; set; }
        public int TicketTypeId { get; set; }
        public bool CurrentValid { get; set; }
    }
}

File: Projekat/src/app/models/PriceList.ts
import { DatePicker } from './DatePicker';
import { DatePipe } from '@angular/common';

export class PriceList{
    Id:number;
    From: string;
    To: string;
    TicketPrice: number;
    PassengerTypeId: number;
    TicketTypeId: number;
    CurrentValid: boolean;

    constructor(private id: number, private from: string, private to:string,private passengerTypeId:number,private ticketPrice:number,private ticketTypeId : number,private currentValid:boolean) { 
        this.Id = id;
        this.From = from;
        this.To = to;
        this.PassengerTypeId = passengerTypeId;
        this.TicketTypeId = ticketTypeId;
        this.TicketPrice = ticketPrice;
        this.CurrentValid = currentValid;
      }

}